avocado/core/test.py

```diff
@@ -213,6 +213,7 @@ def record_and_warn(*args, **kwargs):
 
         self._stdout_file = os.path.join(self.logdir, 'stdout')
         self._stderr_file = os.path.join(self.logdir, 'stderr')
+        self._output_file = os.path.join(self.logdir, 'output')
```

Comment:
@clebergnu would you please add documentation for the `output` file? It's useful, but nobody knows about it (and please include it in release notes)